src/FluentMigrator/IMigrationProcessor.cs

```diff
@@ -29,8 +29,8 @@ public interface IMigrationProcessor : IQuerySchema, IDisposable
         string ConnectionString { get; }
 
         void Execute(string template, params object[] args);
-        DataSet ReadTableData(string schemaName, string tableName);
-        DataSet Read(string template, params object[] args);
+        IDataReader ReadTableData(string schemaName, string tableName);
+        IDataReader Read(string template, params object[] args);
```

Comment:
@marcrocny At this point, I don't think this PR is relevant anymore. Netstandard2.0 reintroduces DataSet, so there's no real reason to go with this.